.eslintrc.js

```diff
@@ -44,7 +44,13 @@ module.exports = {
     "jsx-a11y/label-has-for": "warn", // see #173
     "no-console": "warn",
     "no-extra-semi": "off", // because it isn't that important
-    "react/prop-types": "warn" // see #175
+    "react/prop-types": "warn", // see #175
+    "jsx-a11y/no-onchange": "warn",
+    "jsx-a11y/label-has-for": [ "warn", {
```

Comment:
see above (line 44)

Reply:
This is because even by adding both nesting and htmlForm=id, it does not stop it from giving a warning. I took it out because we are not hitting over 60 warnings with the rest of the proptype validation additions.